app/views/shared/_markdown_help.html.erb

```diff
@@ -18,4 +18,7 @@
 
   <dt><%= t ".image" %></dt>
   <dd>![<%= t ".alt" %>](<%= t ".url" %>)</dd>
+
+  <dt><%= t ".codeblock%></dt>
+  <dd>~~~<br><%= t ".codeblock%><br>~~~</dd>
```

Comment:
These two lines are missing a closing quote for the resource name - it should be `<%= t ".codeblock" %>`.